openmdao/utils/om.py

```diff
@@ -237,14 +241,17 @@ def _view_metamodel(prob):
             view_metamodel(comp, resolution, port_number, browser)
 
         else:
-            try_str = "Try one of the following: {}.".format(mm_names)
+            try_str = "Try one of the following:\n"
```

Comment:
a nicety might be to change the wording if there is only a single mm in the model, e.g.
```
        elif mm_count== 1:
            try_str = ""Did you mean the following:\n"`
        else:
            try_str = "Try one of the following:\n"
```